input_definition.go

```diff
@@ -246,3 +271,28 @@ func (i *InputDefinition) AddFrame(frame InputFrame) error {
 	}
 	return nil
 }
+
```

Comment:
A valid action needs a Frame name and it must be one from the set defined in the frames definition.